style.css

```diff
@@ -310,3 +345,64 @@ body {
   border-radius: 8px;
   color: #fff;
 }
+
+@media only screen and (min-width: 708px) {
+  .work {
+    display: flex;
+    max-width: 70%;
+    margin: auto;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] It would be better to add the margins between project cards to improve your UI design and also to match the Figma design.
- [ ] Following the Figma design, the positions of the image and the project information are switched on the second project card. kindly work on that. 
- Your design:
<img width="703" alt="image" src="https://user-images.githubusercontent.com/77190203/160140236-558af9b6-8468-4da1-a1da-c883c4c90f8e.png">

- Figma design:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/77190203/160140815-b0159379-f582-4f2d-9533-ce0e11d53cf8.png">